Assets/MRTK/Core/Definitions/Devices/DeviceInputType.cs

```diff
@@ -32,6 +32,9 @@ public enum DeviceInputType
         Trigger,
         TriggerTouch,
         TriggerNearTouch,
+        /// <summary>
+        /// TriggerPress maps to the Grab gesture.
```

Comment:
This isn't always the case. It'd be good if we could add consistency here across our mappings, but unfortunately they weren't created that way. I think for now, it might be good to remove this `<summary>`. The other is good!

Reply:
Summary tag removed